README.md

```diff
@@ -2,9 +2,14 @@
 Automates the process of making tennis court reservations for you.
 
 ## Usage
-1. Open `AutoTenez.py` and enter your personal details, and desired time slot(s) to play tennis for tomorrow.
-2. Then run `python3 AutoTenez.py` in a terminal to make the reservation.
+1. Open `AutoTenez.py` and enter your personal details.
+1. Run `python3 AutoTenez.py -h` in a terminal to see all options possible. 
+
+### Example usage
+- `python AutoTenez.py -t 18:30 -f 12341234` attemps to do a reservervation for tomorrow at time `18:30` with friend that has ID `12341234`  
```

Comment:
Let's keep it consistent and use `external reference` instead of `ID`.
```suggestion
- `python AutoTenez.py -t 18:30 -f 12341234` attempt to reserve a time slot for tomorrow at `18:30` with a friend that has external reference `12341234`.
```